datafusion/core/tests/sql/aggregates.rs

```diff
@@ -36,7 +36,7 @@ async fn csv_query_array_agg_distinct() -> Result<()> {
         *actual[0].schema(),
         Schema::new(vec![Field::new_list(
             "ARRAY_AGG(DISTINCT aggregate_test_100.c2)",
-            Field::new("item", DataType::UInt32, false),
```

Comment:
array agg is NULL when there are no inputs, I believe after https://github.com/apache/datafusion/pull/11299 so this change makes sense to me